test_naucse/fixtures/expected-dumps/schema/course-0.0-in.json

```diff
@@ -0,0 +1,187 @@
+$schema: http://json-schema.org/draft-06/schema#
```

Comment:
This is dumped from current? Or do I need to review the schema?

Reply:
It's dumped. The dump is meant to avoid changes to the schema.

Reply:
That's what I expected. Good.